### PR TITLE
Release docker images with predeployed contracts

### PR DESCRIPTION
**Pull request by mpetrun5**
Release docker images with predeployed contracts on tag release.

## Related Issue Or Context

Closes: #19 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

### CODE REVIEW

**Review by MakMuftic — APPROVED** [review]

File: migrations/networks_config.json [hunk 1]
```diff
@@ -14,6 +14,21 @@
...
   },
+  "test2": {
+    "domainID": "2",
+    "erc721Name": "ERC721Test",
+    "erc721Symbol": "ERC721TST",
+    "erc721URI": "metadata.test",
+    "erc20Name": "ERC20Test",
+    "erc20Symbol": "ERC20TST",
+    "erc20LRName": "ERC20Test",
+    "erc20LRSymbol": "ERC20TST",
```
Comment (P1sar) [inline migrations/networks_config.json]: make symbols distinguish

File: truffle-config.js [hunk 2]
```diff
@@ -61,6 +60,12 @@ module.exports = {
       network_id: "*",       // Any network (default: none)
       disableConfirmationListener: true,
     },
+    test2: {
+      host: "127.0.0.1",     // Localhost (default: none)
```
Comment (P1sar) [inline truffle-config.js:64]: are not they suppose to be different?

**Review by P1sar — CHANGES_REQUESTED** [review]

File: migrations/networks_config.json (hunk 1 above)
Comment (mpetrun5) [reply migrations/networks_config.json]: Fixed to match current master.

File: truffle-config.js (hunk 2 above)
Comment (mpetrun5) [reply truffle-config.js:64]: Not really. Both are basically targeting localhost. Just needed to satisfy the form to be able to use the --network param.

**Review by nmlinaric — APPROVED** [review]

**Review by P1sar — APPROVED** [review]

**Pull request merged**